src/ctxcontent/services/analyzer/FightMessagesAnalyzer.js

```diff
@@ -233,16 +278,44 @@ class FightMessagesAnalyzer {
       combatsSums[datePoint].loot[1] += resources?.[1].amount * (accountIsWinner ? 1 : -1);
       combatsSums[datePoint].loot[2] += resources?.[2].amount * (accountIsWinner ? 1 : -1);
 
+      const rounds = JSON.parse(message.querySelector(".rawMessageData").getAttribute("data-raw-combatrounds"));
+      const lastRound = rounds.pop();
+      const losses = {};
+
+      lastRound?.fleets.forEach((side) => {
```

Comment:
one question, if user as no fleet before last round, what happens ?

Reply:
it's good : 
```js
{
                "side": "defender",
                "fleetId": 37414187,
                "technologies":
                [
                    {
                        "technologyId": 210,
                        "destroyed": 0,
                        "destroyedTotal": 5,
                        "remaining": 0
                    }
                ]
            },
```